mothernet/datasets/__init__.py

```diff
@@ -121,7 +147,7 @@ def get_openml_classification(did, max_samples, multiclass=True, shuffled=True):
     return X, y, list(np.where(categorical_indicator)[0]), attribute_names
 
 
-def load_openml_list(dids, filter_for_nan=False, num_feats=100, min_samples=100, max_samples=400,
+def load_openml_list(dids, classification, filter_for_nan=False, num_feats=100, min_samples=100, max_samples=400,
```

Comment:
maybe instead of a multiclass and a classification argument we can have a ``task`` argument that can be ``multiclass``, ``binary`` or ``regression``?